x-pack/plugins/entity_manager/server/lib/entity_client.ts

```diff
@@ -191,13 +192,15 @@ export class EntityClient {
     type,
     start,
     end,
+    sortBy,
```

Comment:
FYI https://github.com/elastic/observability-dev/blob/main/docs/observability-experience/low-level-api-guidelines.md#35-sorting

Reply:
Happy to change, could you point me to a util that helps parsing these strings ?

Reply:
I don't think we have one yet 😅 
But since you're sending this via POST, maybe this JSON format is okay but just rename the parameter? 
I can see how the current format would easily translate to a string **if** we needed to send it as a query parameter instead.